xbmc/dbwrappers/dataset.h

```diff
@@ -31,7 +31,7 @@ class Dataset; // forward declaration of class Dataset
 
 #define DB_CONNECTION_NONE 0
 #define DB_CONNECTION_OK 1
-#define DB_CONNECTION_BAD 2
+#define DB_CONNECTION_DATABASE_NOT_FOUND 2
```

Comment:
Constexpr int? I think we're moving away from defines

Reply:
Okay, will change.